modules/xfem/src/userobjects/MeshCut2DRankTwoTensorNucleation.C

```diff
@@ -175,15 +175,22 @@ MeshCut2DRankTwoTensorNucleation::doesElementCrack(
     point_0 = point_0 - extend_length * crack_dir.unit();
     point_1 = point_1 + extend_length * crack_dir.unit();
 
-    // modify edges of cut should go so that they are on the edge of the domain and have a length
+    // modify edges of cut to put them on the edge of the domain and have a length
     // equal to nucleation_length
     if (is_point_0_on_external_boundary)
     {
+      // first move point back to edge
```

Comment:
@bwspenc this is an error I found.  This is now correct and I ran some a new test case that is in our milestone to check it.  See the second figure in this section:  XFEM Crack Nucleation in the Bulk
Without this commit, the edge cracks were sticking way out of the domain.